services/lightspark_client.go

```diff
@@ -1041,3 +1041,9 @@ func (client *LightsparkClient) getNodeSigningKey(nodeId string) (requester.Sign
 	}
 	return nodeKey, nil
 }
+
+func (client *LightsparkClient) ExecuteGraphql(query string, variables map[string]interface{},
```

Comment:
We actually already have a `ExecuteGraphqlRequest` function on line 761. Do you specifically need the signing key param exposed?

Reply:
OK, great :-D. Feel free to close out this PR if you're good with the existing function

Reply:
sure, thank you